core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,15 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, Long] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.Builder.allTopicPartitions(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    if (response.hasError)
+      throw response.error.exception
+    response.responseData().asScala.map { responseData => (responseData._1, responseData._2.offset) }.toMap
```

Comment:
Will there be any partition error? In the case of all offsets unauthorized partitions are excluded and there won't be any unknown topic partition. Do you think we should throw an exception if there is any?